### PR TITLE
fix: Parse awakened bar & mask correctly [skip ci]

### PR DESCRIPTION
**Pull request by ShadowCat117**
Parse boss bar correctly & reverts mask type to correct mask after the effect expires

https://github.com/Wynntils/Artemis/assets/8337467/e7e3066b-8505-414a-911e-cc06e95c9f4d

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/abilities/bossbars/AwakenedBar.java [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import java.util.regex.Pattern;
 
 public final class AwakenedBar extends TrackedBar {
-    private static final Pattern AWAKENED_PATTERN = Pattern.compile("§fAwakening §7\\[§f(\\d+)/(\\d+)§7]");
+    private static final Pattern AWAKENED_PATTERN = Pattern.compile("§fAwakened §7\\[§f(\\d+)/(\\d+)§7]");
```
Comment (magicus) [inline common/src/main/java/com/wynntils/models/abilities/bossbars/AwakenedBar.java:13]: Just checking, did Wynn change their string or was this always wrong?

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/abilities/bossbars/AwakenedBar.java:13]: There's reports of it not working as far back as July last year so I presume it was changed before then, I've only just leveled a shaman myself so can't confirm when it was changed

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/abilities/bossbars/AwakenedBar.java:13]: This used to work baaack then.

**Review by kristofbolyai — APPROVED** [review]
LGTM

**Pull request merged**